Cargo.toml

```diff
@@ -50,14 +18,12 @@ members = [
   "crates/turborepo-lockfiles",
   "crates/turborepo-scm",
   "crates/turborepo-vercel-api-mock",
+  "crates/utils/*",
```

Comment:
It doesn't make much sense, as auto-hash-map is only used by turbo-tasks, while swc-ast-explorer is its own thing independent of both.

`auto-hash-map` could go into a `turbo-tasks-utils`. `swc-ast-explorer` could be renamed `turbopack-swc-ast-explorer`.

Reply:
Yeah that would be preferable. I think we should avoid having any ambiguous folders, otherwise it'd be too easy for us to accidentally put a turborepo crate in `utils/`

Reply:
Unfortunately, this breaks the glob pattern for `turbopack*` and `turbo-tasks*` above, as `turbo-tasks-utils` wouldn't be a crate, but `turbo-tasks-utils/auto-hash-map` would. The solution here is to keep everything flat and rename them `turbo-tasks-auto-hash-map` and `turbopack-swc-ast-explorer`.

Reply:
Yeah now that I remember, I ran into this exact issue lol. Ah the frustrations of globbing...